src/core/agent/records/notificationRecord.types.ts

```diff
@@ -2,5 +2,9 @@ interface LinkedGroupRequestDetails {
   accepted: boolean;
   saids: Record<string, [string, string][]>;
 }
-
-export type { LinkedGroupRequestDetails };
+interface NotificationAttempts {
+  attempts: number;
+  lastAttempt: number;
+  notificationId : string
```

Comment:
I don't think prettier is running for you with this formatting.

Should be:
```
  saids: Record<string, [string, string][]>;
}

interface NotificationAttempts {
  attempts: number;
  lastAttempt: number;
  notificationId: string;
}

export type { LinkedGroupRequestDetails, NotificationAttempts };
```

I've also asked several times to provide proper whitespace in the code to maintain a consistent code look and feel. This "squashed" view of code where there's no whitespace between anything is messy and hard to follow in large files.